### ISSUE TITLE
questions about quality of SMPLify-X fitting error 13.1mm

### ISSUE THREAD

**Issue by ShirleyMaxx** [mks0601/I2L-MeshNet_RELEASE#80]
Hi, @mks0601,
    Sorry to bother you. I've computed the fitting errors of your provided SMPLify-X fits, on the training set, the error is 25.72mm, and on the test set, the error is 31.24mm, instead of 13.1mm. Could you please help me figure out the reason?
    
1. gt pose3d is from `joint_cam`

2. fitted pose3d is computed by using the `joint_regressor_h36m `with the `mech `coordinates computed by the function `get_smpl_coord `using `shape`, `pose`, and `trans `in your code,

3. I've aligned both to their root. 
    
It seemed that I didn't miss anything? So is it available for you to provide your evaluation code of which get 13.1mm?
    Thank you in advance!

Best,
Shirley

_Originally posted by @ShirleyMaxx in https://github.com/mks0601/I2L-MeshNet_RELEASE/issues/22#issuecomment-885412374_
